Mathlib/Data/Fintype/BigOperators.lean

```diff
@@ -295,4 +295,33 @@ theorem Fintype.prod_sum_type (f : Sum α₁ α₂ → M) :
 #align fintype.prod_sum_type Fintype.prod_sum_type
 #align fintype.sum_sum_type Fintype.sum_sum_type
 
+-- TODO: make sure this is inferred automatically from [Fintype α₁] and [Fintype α₂]
+variable [Fintype (α₁ × α₂)]
```

Comment:
This must be defined in a different file; meaning you need to import something

Reply:
done, imported `Data.Fintype.Prod`